src/common/memusage.c

```diff
@@ -56,7 +56,7 @@
 #include "memusage.h"
 #include "bfind.h"
 
-#if defined(OS_LINUX)
+#if defined(OS_LINUX) || defined(__FreeBSD__)
```

Comment:
Can you please add Solaris as well to this fix?

```suggestion
#if defined(OS_LINUX) || defined(__FreeBSD__) || defined(OS_SOLARIS)
```

Reply:
See: https://github.com/OpenSCAP/openscap/issues/1945

Reply:
I can confirm that the fix works for the Solaris operating system